packages/clarity-visualize/src/layout.ts

```diff
@@ -338,6 +338,14 @@ function setAttributes(node: HTMLElement, data: Layout.DomData): void {
 
     // Add an empty ALT tag on all IMG elements
     if (tag === Constant.ImageTag && !node.hasAttribute(Constant.AltAttribute)) { node.setAttribute(Constant.AltAttribute, Constant.Empty); }
+
+    // Prevent auto fill from working on forms during visualize
```

Comment:
Thanks for fixing this. Can you add a big more about why this is needed (the fact that it looks like the viewer's own auto-complete information is shown in the playback)?